homeassistant/components/devolo_home_control/__init__.py

```diff
@@ -44,6 +37,11 @@ async def async_setup_entry(hass: HomeAssistantType, entry: ConfigEntry) -> bool
 
     gateway_ids = await hass.async_add_executor_job(mydevolo.get_gateway_ids)
 
+    pattern = re.compile(r"\d{16}")
```

Comment:
This can be a constant at the module level.

Reply:
Is moving it to the other constants fine as well?